locksmith/src/operations/wedlocksOperations.ts

```diff
@@ -373,7 +373,7 @@ export const notifyNewKeyToWedlocks = async (
   })
 
   const isAirdroppedRecipient =
-    airdroppedRecipient?.toLowerCase() === ownerAddress?.toLowerCase() &&
+    airdroppedRecipient?.toLowerCase() === ownerAddress?.toLowerCase() ||
     manager?.toLowerCase().trim() ===
       networks[network!]?.keyManagerAddress?.toLowerCase()?.trim()
```

Comment:
Is this intended? We changed this to make sure manager also matches.

Reply:
Yes it is. Since the `manager` is not set correctly right now in the subgraph see https://github.com/unlock-protocol/unlock/issues/11551 that @kalidiagne is working on we need to do an OR.

Once the subgraph has been fixed, we can remove the check on the matching recipient address.